build/targets/AssemblyVersions.props

```diff
@@ -22,7 +24,10 @@
     <FSCoreVersion>4.4.3.0</FSCoreVersion>
     <FSProductVersion>10.1.1.0</FSProductVersion>
     <FSPackageVersion>10.1.4</FSPackageVersion>
-    <VSAssemblyVersion>15.7.0.0</VSAssemblyVersion>
+    <VSMajorVersion>15</VSMajorVersion>
+    <VSMinorVersion>7</VSMinorVersion>
```

Comment:
So, how often do we have to update this?  Will this work for the current 15.8 builds?  Presumably only because we didn't update the assembly version (even though we probably should have)?

Reply:
@pilchie, VS has binding redirects to upgrade references to all of these VS appearances.  We are required to update when we need an API that only exists in the newer versions.  That didn't happen for VS2017.8.  

Staying on the prior reference version is actually pretty handy, because then developers can use the released version of VS to test it.

My guess is we will likely update master, and dev 16. branches to 15.8.0.0 once dev15.8 has been released  for a while.